pkg/volume/volume.go

```diff
@@ -90,7 +90,7 @@ func SetPermissions(mountPath string, mode os.FileMode) error {
 
 func UnmountVolume(mountPath string) error {
 	mounter := mount.New("")
-	return mount.CleanupMountPoint(mountPath, mounter, true)
+	return mounter.Unmount(mountPath)
```

Comment:
Why changing this code? any issue before?

Reply:
We don't really need to remove the mount point. If the mount point is deleted, the healthy check logic will kill the share manager pod. The share manager resource will be marked as `error`. share manager controller can bring it the resource to `running` after reconciliation.

However, to avoid the `error` situation and let share manager controller fully controls the pod life cycle, we can use `mounter.Unmount` which preserves the mount point instead.